xp1/package.json

```diff
@@ -1,10 +1,11 @@
 {
-  "name": "Dust XP1",
+  "name": "dust-xp1",
```

Comment:
Is that OK ?

Reply:
It might be picked up by the final extension packaging but we can figure it out later :+1: